Tasks/DownloadPackageV1/task.json

```diff
@@ -69,7 +68,7 @@
             "properties": {
                 "EditableOptions": "True"
             },
-            "helpMarkDown": "Select the package to download."
+            "helpMarkDown": "If you don't find the package in the list, you can provide the package ID, which you can find using the instructions [here](https://docs.microsoft.com/en-us/rest/api/azure/devops/?view=azure-devops-rest-5.0#get-packages)."
```

Comment:
Can we have a fwdlink instead of hard coded url?

Reply:
Fixed